docs/testing.md

```diff
@@ -16,6 +16,15 @@
 The base Docker installation methods for popular Linux distributions are listed here:
 	https://docs.docker.com/engine/install/
 
+To run Docker commands without **sudo**, add your user to **docker** group
```

Comment:
I am not sure that it is valid for all host systems (name of the group might be different but it is just a guess). Please double-check in docker docs.

Reply:
Yes, it is valid. I added the link to official page as a reference for all post-installation procedures.